client/app/scripts/controllers/repos.js

```diff
@@ -76,6 +76,13 @@ angular.module('clientApp')
 
             $scope.repos[key].shoov_id = data.repository;
             $scope.repos[key]._inProgress = false;
+          }, function(response) {
+            if (response.data.title == 'no_config_file') {
```

Comment:
@HelenaEksler is this still correct? I was expecting to see the red warning in the UI - but I think it's not in the HTML in this PR.